app/models/solid_queue/queue.rb

```diff
@@ -40,6 +40,19 @@ def size
       @size ||= ReadyExecution.queued_as(name).count
     end
 
+    def latency
+      @latency = begin
+        now = Time.current
+        oldest_enqueued_at = SolidQueue::ReadyExecution.queued_as(name).minimum(:created_at) || now
```

Comment:
```suggestion
        oldest_enqueued_at = ReadyExecution.queued_as(name).minimum(:created_at) || now
```
Just for consistency with the other methods.